### PR TITLE
add prodigy9

### PR DESCRIPTION
**Pull request by matthewmayer**
## What happened

Add Prodigy9 as another silver sponsor.
 
## Insight

Now there are 3 silver and 3 gold its more balanced so can remove some of the alignment hacks for silver.

https://deploy-preview-217--bangkokrb-rubyconfth.netlify.app/
 
## Proof Of Work

Screenshot 2022-10-14 at 09 45 41
Screenshot 2022-10-14 at 09 46 18

### CODE REVIEW

File: src/_data/sponsors.yml [hunk 1]
```diff
@@ -23,3 +23,8 @@
   logo_url: /images/sponsors/oivan.svg
   url: "https://oivan.com/"
   level: gold
+- name: PRODIGY9
+  about: "Better people, better software."
+  url: https://www.cloud66.com/frameworks/rails/?utm_source=e&utm_medium=l&utm_campaign=RubyConfTH
+  logo_url: /images/sponsors/prodigy9.svg
+  level: silver
```
Comment (citizen428) [inline src/_data/sponsors.yml:30]: Do you want to add the missing newline Github is complaining about?

**Review by citizen428 — APPROVED** [review]

File: src/_data/sponsors.yml (hunk 1 above)
Comment (matthewmayer) [reply src/_data/sponsors.yml:30]: Where is it complaining?

Comment (citizen428) [reply src/_data/sponsors.yml:30]: Right at the end of the file in the diff view:

Screen Shot 2022-10-14 at 23 43 10

Comment (olivierobert) [reply src/_data/sponsors.yml]: I did the simple fix

**Review by olivierobert — APPROVED** [review]

**Pull request merged**